assignment08/index.php

```diff
@@ -0,0 +1,29 @@
+
+<?php
+    include "includes/header.php";
+    $my_weight= '125 lb';
+    $_age = 25;
+    $first_name='Donjeta';
+    $last_name='Emerllahu';
+    $poundsToKg= 'pounds *(1.2.2)';
+    //write PHP here...
+   function poundsToKg($my_weight){
+       $my_weight = ("pounds *(1/2.2");
```

Comment:
this will not convert your weight to kg. You would need something like:
$my_weight = $my_weight * (1/2.2);